tests/requirements-test-centos7.txt

```diff
@@ -1,2 +1,2 @@
 # CentOS 7
-setuptools==0.9.8
+setuptools
```

Comment:
Why the unpin here?

Reply:
The tests for CentOs for Python3 were failing with:
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bazel-out/host/bin/compiler/compiler.par/__main__.py", line 19, in <module>
  File "bazel-out/host/bin/compiler/compiler.par/subpar/runtime/support.py", line 338, in setup
  File "bazel-out/host/bin/compiler/compiler.par/subpar/runtime/support.py", line 169, in _setup_pkg_resources
  File "/home/ubuntu/ws/sdmain/subpar/.env/lib/python3.6/site-packages/pkg_resources.py", line 1545, in <module>
    register_loader_type(importlib_bootstrap.SourceFileLoader, DefaultProvider)
AttributeError: module 'importlib._bootstrap' has no attribute 'SourceFileLoader'
```

The current version for Python3 seems to be https://centos.pkgs.org/7/centos-x86_64/python3-setuptools-39.2.0-10.el7.noarch.rpm.html. So I unpinned the version for it to pick it up accordingly based on the python version.

Python2 setuptools version is https://centos.pkgs.org/7/centos-x86_64/python-setuptools-0.9.8-7.el7.noarch.rpm.html